inc/settings.php

```diff
@@ -77,16 +77,16 @@ function create_policy_page() {
 	if (
 		// Validate the nonce.
 		! isset( $_POST['_altis_privacy_policy_page_nonce'] ) ||
-		! wp_verify_nonce( sanitize_text_field( $_POST['_altis_privacy_policy_page_nonce'] ), 'altis.privacy_policy_page' ) ||
+		! wp_verify_nonce( sanitize_text_field( wp_unslash( $_POST['_altis_privacy_policy_page_nonce'] ) ), 'altis.privacy_policy_page' ) ||
 		// Make sure we've requested a new policy page.
 		! isset( $_POST['create_policy_page'] ) ||
 		// Make sure that the request was valid.
-		! in_array( esc_attr( $_POST['create_policy_page'] ), get_allowed_policy_page_values(), true )
+		! in_array( sanitize_text_field( wp_unslash( $_POST['create_policy_page'] ) ), get_allowed_policy_page_values(), true )
 	) {
 		return;
 	}
 
-	$policy_page = esc_attr( $_POST['create_policy_page'] );
+	$policy_page = sanitize_text_field( wp_unslash( $_POST['create_policy_page'] ) );
```

Comment:
There are multiple calls to this `$_POST` var so we could assign it to a variable earlier. Not needed for this PR though.

Reply:
Created a ticket to take care of those: https://github.com/humanmade/altis-consent/issues/31